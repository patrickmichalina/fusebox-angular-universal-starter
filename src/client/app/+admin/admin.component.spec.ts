import { AdminComponent } from './admin.component';
import { async, ComponentFixture, TestBed } from '@angular/core/testing';
import { Component } from '@angular/core';
import { AdminModule } from './admin.module';

describe(AdminComponent.name, () => {
  let fixture: ComponentFixture<AdminComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [AdminModule],
      declarations: [TestComponent]
    }).compileComponents();
  }));

  beforeEach(async(() => {
    fixture = TestBed.createComponent(AdminComponent);
  }));

  afterEach(async(() => {
    TestBed.resetTestingModule();
  }));

  it('should match snapshot', async(() => {
    expect(fixture).toMatchSnapshot();
  }));

  it('should compile', async(() => {
    expect(fixture.nativeElement).toBeDefined();
  }));
});

@Component({
  selector: 'test-component',
  template: '<pm-admin></pm-admin>'
})
class TestComponent {}
