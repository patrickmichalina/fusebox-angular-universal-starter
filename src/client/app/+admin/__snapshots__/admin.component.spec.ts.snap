// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminComponent should match snapshot 1`] = `
<pm-admin>
  <md-card
    class="mat-card"
  >
    
    
    <h1>
      Admin
    </h1>
    
    
    <p>
      Your Admin module is routed here
    </p>
    
  
  </md-card>
</pm-admin>
`;
